add_package.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: packages
+  serial: 1
+  gather_facts: no
+  vars:
+    gem_name: "{{ inventory_hostname.replace('tfm-ror51-rubygem-', '') }}"
+  roles:
+    - git-annex-setup
+  tasks:
+    - name: 'Create package directory'
+      file:
+        state: directory
+        path: "packages/{{ inventory_hostname }}"
+
+    - name: 'Fetch gem'
+      command: "gem fetch {{ gem_name }} --version {{ version }}"
+      args:
+        chdir: "packages/{{ inventory_hostname }}"
+
+    - name: 'Git annex add new file(s)'
+      command: "git annex add {{ gem_name }}"
+      args:
+        chdir: "packages/{{ inventory_hostname }}"
+
+    - name: 'Run gem2rpm'
+      shell: "gem2rpm -t ../../scl-rubygem-GEM.spec.template {{ gem_name }}-{{ version }}.gem > {{ inventory_hostname }}.spec"
```

Comment:
This may not be needed that much, but what do you think of making the template a var too?

Reply:
What may not be needed that much?

Reply:
Moving the name of the template to a separate variable.